src/enums.h

```diff
@@ -245,5 +245,15 @@ typedef enum {
 } htcp_clr_reason;
 #endif /* USE_HTCP */
 
+typedef enum {
+#if USE_OPENSSL
+    xaStepTlsBump1,
+    xaStepTlsBump2,
+    xaStepTlsBump3,
+#endif
+    xaStepGeneratingConnect,
```

Comment:
Would it be possible to make xaStepGeneratingConnect value _stable_ and all configured/set step values positive by placing xaStepGeneratingConnect first (before `#if/#endif`) and setting it to 1? Adding a zero xaStepUnknown or similar may also be a good idea. Or do we have to preserve `xaStepTlsBump{1,2,3}` absolute values?

Reply:
commit ccd68c4b68
No need to preserve 1 2 and 3 values for tls bumping modes.